client/src/substack.scss

```diff
@@ -13,17 +13,11 @@
 .home-sitemap{
   display: none!important;
 }
-//.poll-embed{
-//  display: none!important;
-//}
 .subscribe-footer{
   display: none!important;
 }
-.portable-archive.empty-list{
-  display: none!important;
-}
 .subscribe-iframe{
-  height: 100px;
+  display: none!important;
```

Comment:
might need this back because we still need a way for people to subscribe. I will check locally